tests/pytorch/fused_attn/run_fused_attn_with_cp.py

```diff
@@ -167,13 +176,6 @@ def run_dpa_with_cp(
     else:
         bias = None
 
-    # make sure all GPU ranks have same inputs
-    for x in [q, k, v, dout] + ([] if bias is None else [bias]):
-        dist.broadcast(x, 0, group=cp_comm_group)
-    if qkv_format == "thd":
-        for x in [cu_seqlens_q, cu_seqlens_q_padded, cu_seqlens_kv, cu_seqlens_kv_padded]:
-            dist.broadcast(x, 0, group=cp_comm_group)
-
```

Comment:
This is because attention test has [random seed setting](https://github.com/xrennvidia/TransformerEngine/blob/xren/cp_a2a_p2p/tests/pytorch/fused_attn/test_fused_attn.py#L56-L61), so all GPUs should have same randomly generated inputs, the broadcast is redundant.